src/styles/app.scss

```diff
@@ -125,3 +125,7 @@
 #release-head {
   font-family: 'Roboto', sans-serif;
 }
+
+.modal-backdrop {
+  background-color: #7e8081 !important;
+}
```

Comment:
@abulojoshua1 Why are you using bootstrap class to change this. Don't you think that this can affect other parts of the app where this class was used?

Reply:
@muhozi, correct me if I am wrong, my goal of this task was to not only make the `subscription modal` background lighter but to also make all modal background colors consistent.